### PR TITLE
Changelog v2.7.0 + Bump version

### PR DESCRIPTION
**Pull request by ddeidda**
Signed-off-by: ddeidda <ddeidda@amazon.com>

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -2,6 +2,41 @@
...
+
+**ENHANCEMENTS**
+
+* FSx Lustre: Exit with error when failing to retrieve FSx mountpoint
+* `sqswatcher`: The daemon is now compatible with VPC Endpoints so that SQS messages can be passed without traversing
+  the public internet.
+
+**CHANGES**
+
+* Upgrade Nice DCV to version 2020.0-8428.
```
Comment (enrico-usai) [inline CHANGELOG.rst:16]: The name of the product is NICE DCV.

File: CHANGELOG.rst [hunk 2]
```diff
@@ -2,6 +2,41 @@
...
+* Upgrade Intel MPI to version U7.
+* Upgrade NVIDIA driver to Tesla version 440.64.00.
```
Comment (mauri-melato) [inline CHANGELOG.rst:18]: I'd avoid to call out 'Tesla'

File: CHANGELOG.rst [hunk 3]
```diff
@@ -2,6 +2,41 @@
...
+* Upgrade EFA installer to version 1.8.4:
+
+  * Kernel module: ``efa-1.5.1`` (no change)
+  * RDMA core: ``rdma-core-25.0`` (no change)
+  * Libfabric: ``libfabric-aws-1.9.0amzn1.1`` (no change)
+  * Open MPI: openmpi40-aws-4.0.3 (updated from openmpi40-aws-4.0.2)
+
+* Download and install Chef client from ParallelCluster's S3 buckets rather than from Chef website when building an AMI.
```
Comment (mauri-melato) [inline CHANGELOG.rst:26]: We can remove this

File: CHANGELOG.rst [hunk 4]
```diff
@@ -2,6 +2,41 @@
...
+
+* Configuration -
+
+  * scheduler parameter: This parameter no longer has a default value and is now mandatory. A deprecation warning
```
Comment (mauri-melato) [inline CHANGELOG.rst:30]: ``base_os`` and ``scheduler`` parameters are now mandatory and they have no longer a default value.
Please remove the part about the warning message.

File: CHANGELOG.rst [hunk 5]
```diff
@@ -2,6 +2,41 @@
...
+    is now shown if ``sge`` or ``torque`` are set as value.
+  * base_os parameter: New default value is ``alinux2``
```
Comment (mauri-melato) [inline CHANGELOG.rst:32]: see above

File: CHANGELOG.rst [hunk 6]
```diff
@@ -2,6 +2,41 @@
 CHANGELOG
 =========
 
+2.7.0
+=====
+
+**ENHANCEMENTS**
+
+* FSx Lustre: Exit with error when failing to retrieve FSx mountpoint
```
Comment (mauri-melato) [inline CHANGELOG.rst:10]: This is a fix

File: CHANGELOG.rst (hunk 2 above)
Comment (ddeidda) [reply CHANGELOG.rst:18]: It's called out in past upgrades as well, f.i. in 2.5.1. Do you think we should stop doing it?

File: CHANGELOG.rst [hunk 7]
```diff
@@ -2,6 +2,35 @@
...
+  * RDMA core: ``rdma-core-25.0`` (no change)
+  * Libfabric: ``libfabric-aws-1.9.0amzn1.1`` (no change)
+  * Open MPI: openmpi40-aws-4.0.3 (updated from openmpi40-aws-4.0.2)
+
+* Configuration: base_os and scheduler parameters are now mandatory and they have no longer a default value.
+
+**BUG FIXES**
+
+* Fix recipes installation at runtime by adding the bootstrapped file at the end of the last chef run.
+* Solved performance issues with Centos 7
```
Comment (mauri-melato) [inline CHANGELOG.rst:30]: Please remove this.

File: CHANGELOG.rst (hunk 2 above)
Comment (mauri-melato) [reply CHANGELOG.rst:17]: yes

File: CHANGELOG.rst [hunk 8]
```diff
@@ -2,6 +2,35 @@
...
 
+2.7.0
+=====
+
+**ENHANCEMENTS**
+
+* `sqswatcher`: The daemon is now compatible with VPC Endpoints so that SQS messages can be passed without traversing
+  the public internet.
+
+**CHANGES**
```
Comment (mauri-melato) [inline CHANGELOG.rst:13]: Le'ts add an entry for   
 Upgrade CentOS 7 AMI that includes kernel 7.8

**Pull request merged**